python/cudf/cudf/tests/test_udf_masked_ops.py

```diff
@@ -251,7 +255,7 @@ def test_apply_mixed_dtypes(dtype_a, dtype_b, op):
     # First perform the op on two dummy data on host, if numpy can
     # safely type cast, we should expect it to work in udf too.
     try:
-        op(getattr(np, dtype_a)(0), getattr(np, dtype_b)(42))
+        op(getattr(np, left_dtype)(0), getattr(np, right_dtype)(42))
```

Comment:
I think this is a potential alternative to using `getattr`, if you prefer it:
```suggestion
        op(np.dtype(left_dtype).type(0), np.dtype(right_dtype).type(42))
```